framework/components/ecl-forms/ecl-forms-legends/_legends.scss

```diff
@@ -1,13 +1,10 @@
-/*
-Legend
+/**
+* Legend component
+* @define form-legend
 */
 
 $form-item-legend-font-size: map-get($ecl-font-size, 'xs');
 
-.legend {
+.ecl-form-legend {
```

Comment:
not sure that we have to name class "ecl-form-legend", and not only "ecl-legend". For instance we name other component's classes "ecl-text-input" or "ecl-select"